tools/saqc/saqc.xml

```diff
@@ -883,327 +782,409 @@
+                  <option value="type_0">Selection</option>
+                  <option value="type_1">List of String</option>
                 </param>
-                <when value="valid">
-                  <param name="history" type="hidden" value="valid" label=""/>
+                <when value="type_0">
+                  <param argument="history" type="select" optional="true" label="Discriminate the plotted flags with respect to the tests they originate from" help="*  valid : Only plot flags, that are not overwritten by subsequent tests. Only list tests in the legend, that actually contributed flags to the overall result. * None: Just plot the resulting flags for one variable, without any historical and/or meta information. * list of strings: List of tests. Plot flags from the given tests, only. * complete (not recommended, deprecated): Plot all the flags set by any test, independently from them being removed or modified by subsequent modifications. (this means: plotted flags do not necessarily match with flags ultimately assigned to the data)">
+                    <option value="valid">valid</option>
+                    <option value="complete">complete</option>
+                  </param>
                 </when>
-                <when value="complete">
-                  <param name="history" type="hidden" value="complete" label=""/>
+                <when value="type_1">
+                  <param argument="history" type="text" label="Discriminate the plotted flags with respect to the tests they originate from" help="*  valid : Only plot flags, that are not overwritten by subsequent tests. Only list tests in the legend, that actually contributed flags to the overall result. * None: Just plot the resulting flags for one variable, without any historical and/or meta information. * list of strings: List of tests. Plot flags from the given tests, only. * complete (not recommended, deprecated): Plot all the flags set by any test, independently from them being removed or modified by subsequent modifications. (this means: plotted flags do not necessarily match with flags ultimately assigned to the data)" multiple="true"/>
                 </when>
-                <when value="list">
-                  <param argument="history" type="text" value="" label="Discriminate the plotted flags with respect to the tests they originate from (comma-separated)">
-                    <validator type="empty_field"/>
+              </conditional>
+              <conditional name="xscope_cond">
+                <param name="xscope_selector" type="select" label="Choose type for 'Determine a chunk of the data to be plotted'" help="The parameter supports different input formats, you can choose which one suites your application.">
+                  <option value="type_0">slice</option>
+                  <option value="type_1">OffsetStr (Pandas Frequency)</option>
+                  <option value="type_2">String</option>
```

Comment:
Wondering if we should drop the `str` option in these cases. In saqc's python API one can give an object of type `slice`, `OffsetStr`, or a `str` (which is then converted to slice or `OffsetStr`, isn't it?)

Here we can only give strings formated in the slice or offsetstr format, ie. str seems redundant?